File: 1º Semestre/Progr Comp 1/aula6/ex2.c
// Desenvolva um programa em C que solicite ao usuário informa um número inteiro. 
// O programa em C deverá mostrar na tela os números de 1 a N (número inteiro informado). 
// Exemplo: se o usuário informar o número 11, o programa em C deverá mostrar na tela os números 1,2,3,4,5,6,7,8,9,10 e 11.

#include <stdio.h>

int main () {
    int n,x;

    printf("Digite um numero inteiro: ");
    scanf("%d",&n);

    for (x=1;x<=n;x++) {
        printf("%d",x);
    }

    return 0;
}

File: 1º Semestre/Introd Comp e Tec/aula2/ex2.py
# Desenvolva um programa em Python que solicite ao usuário digitar a idade de uma pessoa.
# Após a digitação, o computador deverá verificar se esta pessoa é maior ou menor de idade e mostrar esta informação na tela do computador.
# Exemplos: se o usuário informar que a pessoa tem 17 anos, deverá ser mostrada na tela a mensagem “Pessoa menor de idade”. 
# Se o usuário informar que a pessoa tem 18 anos, deverá ser mostrada na tela a mensagem “Pessoa maior de idade”.

idade = int(input("Digite a idade: "))

if idade >= 18:
    print("Pessoa maior de idade")
else:
    print("Pessoa menor de idade")